src/js/interim.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * The following theme has transitional properties to
+ * mitigate the upgrade to the design-system theme.
```

Comment:
I think we should be specific about migrating from v1 to v2 of the theme.

Reply:
Done.